test_enhancements_simple.py

```
@@ -0,0 +1,179 @@
+    print("Testing error handling...")
+    
+    # Test isexception function
+    assert records.isexception(ValueError("test")) == True
+    assert records.isexception(ValueError) == True
+    assert records.isexception("string") == False
+    assert records.isexception(42) == False
+    
+    # Test invalid database URL
+    try:
+        records.Database(None)
+        assert False, "Should have raised ValueError"
```

Ruff (0.14.0)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)